### ISSUE TITLE
AdBlocker Issue

### ISSUE THREAD

**Issue by thekendog** [kslimani/clappr-ima-plugin#30]
Certain AdBlockers are causing issues with the plugin.  Privacy Badger on Firefox is one.  When it's enabled the ad is obviously blocked but the video player shows a frame from the video and no controls.  It's essentially a frozen image of the video/stream with no controls.

**Comment by kslimani** [comment]
Can you reproduce the issue with https://kslimani.github.io/clappr-ima-plugin/ ? (please provide also device/browser/os versions)

**Comment by thekendog** [comment]
Yeah try it on Firefox with the Privacy Badger extension.  In my testing I was able to figure out that I had to set requestAdIfNoAutoplay to false.
